client/src/components/LoginSignupContainer.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Paper } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles({
+  root: {
+    background: "linear-gradient(to bottom, #43DDC1, #501CBD)",
```

Comment:
add these variables to the color in the theme (in palette) and then use a function in makeStyles like this:
https://material-ui.com/styles/basics/#stress-test
And then use a template string for the linear-gradient to use the varialbes